configs/application/config.json

```diff
@@ -51,14 +52,21 @@
 					"Toolbar",
 					"App Launcher",
 					"Workspace Management Menu",
-					"Finsemble Linker Window",
+					"linkerWindow",
```

Comment:
Moved all of these components out of the default agent so they won't show up in "simple view", which hides the system components process.